### PR TITLE
Dockerfile: Update bitcoin-core to 0.16.0

### PR DESCRIPTION
**Pull request by jsarenik**
This is my take on update for ElementsProject/lightning#1127

### CODE REVIEW

**Comment by jsarenik** [comment]
Sorry, I did not manage to test it. Maybe tomorrow.

**Comment by jsarenik** [comment]
Maybe it should go straight to master on `ElementsProject/lightning` but I was mistaken by the name of docker image...

File: contrib/Dockerfile.builder [hunk 1]
```diff
@@ -30,10 +30,11 @@ RUN apt-get -qq update && \
 	rm -rf /var/lib/apt/lists/*
 
 RUN cd /tmp/ && \
-    wget https://bitcoin.org/bin/bitcoin-core-0.15.1/bitcoin-0.15.1-x86_64-linux-gnu.tar.gz -O bitcoin.tar.gz && \
+    VER=0.16.0 && \
```
Comment (cdecker) [inline contrib/Dockerfile.builder]: Pulling this out into a container wide env-var might be a good idea.

**Comment by jsarenik** [comment]
@cdecker see the updated commit please

**Pull request closed without merging**